### PR TITLE
Added Domain feature to feature-permissions.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2394.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/permissions-exo/feature-permissions.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ You can use the features in the following table to manage your Exchange Online o
 |Data loss prevention (DLP)|Organization Management <br/><br/> Compliance Management|
 |Discovery mailboxes - Create|Organization Management <br/> Recipient Management|
 |Distribution groups|Organization Management <br/> Recipient Management|
+|Domains|Organization Management|
 |Microsoft 365 or Office 365 connectors|Organization Management|
 |In-Place eDiscovery|Discovery Management  <br/> **Note**: By default, the Discovery Management role group doesn't have any members. No users, including adminis, have the required permissions to search mailboxes. For more information, see [Assign eDiscovery permissions in Exchange](../security-and-compliance/in-place-ediscovery/assign-ediscovery-permissions.md).|
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/permissions-exo/feature-permissions.md:55]:
```suggestion
|In-Place eDiscovery|Discovery Management  <br/> **Note**: By default, the Discovery Management role group doesn't have any members. No users, including admins, have the required permissions to search mailboxes. For more information, see [Assign eDiscovery permissions in Exchange](../security-and-compliance/in-place-ediscovery/assign-ediscovery-permissions.md).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - unrelated, but please also correct the typo in Line 55.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**